applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -175,6 +175,14 @@ static void stream_recv_cb(struct bt_audio_stream *stream, const struct bt_iso_r
 		bad_frame = true;
 	}
 
+	if (buf->len != active_stream.codec->octets_per_sdu) {
```

Comment:
To me it seems cleaner to do bt_codec_cfg_get_octets_per_frame(stream->codec) here. In other words use the stream variable inputted to this callback

Reply:
Changed